zfs/mapping.go

```diff
@@ -87,5 +97,9 @@ func ZFSListMappingProperties(ctx context.Context, filter DatasetFilter, propert
 
 	}
 
+	jobid := zfscmd.GetJobIDOrDefault(ctx, "__nojobid")
+	metric := prom.ZFSListUnmatchedUserSpecifiedRulesCount.WithLabelValues(jobid)
+	metric.Set(float64(len(unmatchedUserSpecifiedDatasets)))
```

Comment:
This needs to be a counter that is incremented using `.Add()`.

The reason is that, at the zfs package level, we don't that the endpoint.go:ListFilesystems function is the only caller.

So, if we just `Set()` it to whatever the latest result was, it could flip if there are two callers, one with tan unmatched dataset, one without.

The solution is to use `Add()` with a counter, and use Prometheus to alert on "did the counter increase in the last $timeframe".